languages/python/oso/oso.py

```diff
@@ -37,7 +37,7 @@ def __init__(self):
         self.register_class(PathMapper)
 
     # TODO (dhatch): should we name this 'is_allowed'?
```

Comment:
```suggestion
```

Reply:
Finally! That TODO has been there for so long.